hiera.yaml

```diff
@@ -4,4 +4,5 @@ version: 5
 hierarchy:
   - name: OS family
     paths:
+      - '%{facts.os.family}.%{facts.os.release.major}.yaml'
```

Comment:
As filename is`RedHat9.yaml`, need to lose the `.` here ?

```suggestion
      - '%{facts.os.family}%{facts.os.release.major}.yaml'
```

Reply:
Thinking it through, Debian et all might have their own.  So I switched to a subdir way to keep things sorted if that happens.